user-api/overlays/test/deploymentconfig.yaml

```diff
@@ -1,197 +0,0 @@
----
```

Comment:
can please also remove the deployment config from here: https://github.com/thoth-station/thoth-application/blob/e0af5c68dd95869e43ee950ca4aae74b1cc26b31/user-api/overlays/test/kustomization.yaml#L13

Reply:
thanks!! nice catch!